qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/QuteCodeActions.java

```diff
@@ -298,4 +332,137 @@ private static void doCodeActionsForUndefinedSectionTag(Template template, Diagn
 			LOGGER.log(Level.SEVERE, "Creation of undefined user tag code action failed", e);
 		}
 	}
+
+	private static void doCodeActionsForSimilarValues(Template template, Diagnostic diagnostic,
+			List<CodeAction> codeActions, JavaDataModelCache javaCache, String varName) throws BadLocationException {
+		int offset = template.offsetAt(diagnostic.getRange().getStart());
+		Node node = QutePositionUtility.findBestNode(offset, template.findNodeAt(offset));
+
+		Range diagnosticRange = diagnostic.getRange();
+
+		Range rangeValue = new Range(

```

Comment:
I'm pretty sure the range value is off.

![similar-values-bug](https://user-images.githubusercontent.com/1417342/179018098-3e44a34d-ec78-4438-9be3-13331ea845d6.gif)